fuzz/ada_fuzzer.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+dict = url.dict
```

Comment:
Can you add ending missing newlines to each of these files? editorconfog extension for your ide should fix this

Reply:
hmm, will update it. what ide do you use?

Reply:
zed and clion